Source/WebInspectorUI/UserInterface/Views/LayoutDetailsSidebarPanel.js

```diff
@@ -131,6 +131,11 @@ WI.LayoutDetailsSidebarPanel = class LayoutDetailsSidebarPanel extends WI.DOMDet
         showSectionIfNotEmpty(this._flexNodesSection, this._flexNodesDetailsSectionRow, this._flexNodeSet);
     }
 
+    supportsDOMNode(nodeToInspect)
+    {
+        return nodeToInspect.nodeType() === Node.ELEMENT_NODE;
```

Comment:
please add a `&& super.supportsDOMNode(nodeToInspect)` so that if needed we can change the parent class implementation and have it affect all subclasses